code/modules/mechs/components/body.dm

```diff
@@ -254,9 +237,11 @@
 	icon_state = "light_body"
 	max_damage = 75
 	power_use = 5
+	emp_shielded = TRUE
```

Comment:
Also not communicated.

Additionally, consider this reducing EMP severity by 1 rather than straight up blocking it.

Reply:
I'll change the description, but it's supposed to replicate the job of mecha plating e.g. blocks it completely but has awful resistances against other damage sources, this is actually worse because it now doesn't block laser/plasma. Unless you'd like for combat mechs to be partially resistant against EMP while also have 'energy' armor.

Reply:
Before: heavy components, light EMP armour (that had high resists against lasers/plasma). Now: light/fragile components, piss poor armour against everything except EMP